install/seurcashondelivery/seurcashondelivery.php

```diff
@@ -307,7 +306,36 @@ public function hookPaymentReturn($params)
 				return $this->display(__FILE__, 'views/templates/hook/confirmation.tpl');
     }
 
-	public function getCargo($cart, $check_id_currency = true)
+	public function getCargo($order, $check_id_currency = true)
+	{
+
+		if ((int)$order->id >0)
+		{
+			$sql = 'SELECT codfee FROM '._DB_PREFIX_.'seur_order WHERE id_order='.$order->id;
```

Comment:
You cast the variable in your test, but not in the SQL request. Please add a `(int)` cast